src/test/resources/day5/input1.in

```diff
@@ -0,0 +1,500 @@
+457,98 -> 457,981
+277,707 -> 277,531
+943,721 -> 37,721
+959,295 -> 702,295
+23,547 -> 891,547
+209,114 -> 931,836
+737,174 -> 737,195
+208,890 -> 115,797
+170,401 -> 726,401
+11,218 -> 11,297
+989,10 -> 10,989
+866,86 -> 487,86
+867,31 -> 867,334
+846,414 -> 861,414
+478,315 -> 478,697
+572,843 -> 731,843
+657,12 -> 161,508
+903,194 -> 142,955
+612,321 -> 147,786
+813,920 -> 259,920
+834,389 -> 651,206
+824,153 -> 824,557
+399,871 -> 115,871
+270,785 -> 270,120
+469,640 -> 753,640
+620,132 -> 620,175
+620,234 -> 666,234
+594,409 -> 948,55
+670,323 -> 670,89
+262,65 -> 262,379
+879,617 -> 284,22
+493,423 -> 761,423
+17,931 -> 906,42
+512,494 -> 473,494
+122,230 -> 122,87
+15,207 -> 533,207
+216,183 -> 50,183
+360,107 -> 280,107
+403,841 -> 941,841
+913,442 -> 500,29
+864,947 -> 864,85
+500,516 -> 634,382
+283,20 -> 669,20
+916,770 -> 176,30
+966,73 -> 252,787
+847,841 -> 171,165
+163,219 -> 766,219
+482,515 -> 275,308
+528,949 -> 240,949
+725,574 -> 847,696
+109,131 -> 109,538
+655,837 -> 476,837
+803,631 -> 803,51
+977,83 -> 149,911
+207,231 -> 171,231
+617,29 -> 617,294
+838,708 -> 446,708
+711,597 -> 612,498
+975,942 -> 279,246
+315,128 -> 315,293
+146,962 -> 873,235
+448,180 -> 54,180
+177,680 -> 866,680
+891,265 -> 741,265
+656,949 -> 414,949
+909,456 -> 196,456
+574,286 -> 58,286
+861,691 -> 861,383
+779,351 -> 779,827
+459,989 -> 459,350
+936,480 -> 936,699
+645,309 -> 348,606
+861,62 -> 621,302
+568,324 -> 568,358
+889,221 -> 889,335
+538,759 -> 538,266
+780,736 -> 780,827
+866,518 -> 983,401
+67,871 -> 840,98
+432,664 -> 664,664
+146,24 -> 755,24
+964,585 -> 964,770
+372,144 -> 809,144
+688,827 -> 867,827
+137,916 -> 137,942
+846,131 -> 846,46
+764,21 -> 457,328
+140,66 -> 799,725
+703,224 -> 83,844
+557,67 -> 557,681
+355,544 -> 764,135
+625,893 -> 126,394
+842,214 -> 842,322
+582,778 -> 582,762
+341,861 -> 341,859
+143,767 -> 52,858
+114,109 -> 114,200
+394,210 -> 396,212
+861,353 -> 861,652
+873,553 -> 62,553
+44,962 -> 984,22
+734,56 -> 734,828
+798,516 -> 950,516
+367,755 -> 367,618
+868,637 -> 868,780
+192,952 -> 192,734
+603,109 -> 705,211
+12,17 -> 984,989
+910,147 -> 910,620
+515,749 -> 515,517
+775,136 -> 761,150
+662,636 -> 662,21
+894,490 -> 310,490
+956,732 -> 297,73
+514,99 -> 140,99
+308,419 -> 691,419
+485,86 -> 485,187
+737,783 -> 979,783
+90,76 -> 869,855
+959,112 -> 84,112
+879,494 -> 879,257
+933,425 -> 933,619
+64,391 -> 64,21
+106,305 -> 253,452
+324,152 -> 853,152
+666,225 -> 39,852
+370,904 -> 257,791
+592,845 -> 592,15
+936,971 -> 267,302
+147,210 -> 62,210
+308,323 -> 495,323
+212,918 -> 110,918
+229,392 -> 685,848
+896,132 -> 326,702
+483,143 -> 605,265
+251,317 -> 130,317
+758,93 -> 445,93
+156,286 -> 458,286
+401,904 -> 383,904
+244,256 -> 851,256
+928,411 -> 612,411
+642,920 -> 642,420
+494,707 -> 494,225
+87,112 -> 87,256
+972,907 -> 83,18
+139,104 -> 139,761
+493,725 -> 493,529
+981,145 -> 459,667
+390,240 -> 702,240
+466,982 -> 807,982
+320,143 -> 692,515
+477,649 -> 477,206
+456,254 -> 456,578
```

Comment:
Tutaj jako ciekawostkę podam fakt, że dane wejściowe z czasem się zmieniają :) Dane dla których znalazłem rozwiązanie różnią się od Twoich, a zabrałem się za to zadanie z opóźnieniem kilkudniowym :D

Reply:
w ogóle czytałem ostatnio na Reddicie (polecam bardzo subreddit adventu btw! https://www.reddit.com/r/adventofcode/) że każdy dostaje inny input żeby nie było ściągania odpowiedzi :D